crates/vault-registry/src/types.rs

```diff
@@ -158,27 +161,6 @@ pub struct Vault<AccountId, BlockNumber, Wrapped, Collateral, SignedFixedPoint>
     /// Amount of collateral that is locked for remaining to_be_redeemed
     /// tokens upon liquidation.
     pub liquidated_collateral: Collateral,
-    /// Block height until which this Vault is banned from being used for
-    /// Issue, Redeem (except during automatic liquidation) and Replace.
-    pub banned_until: Option<BlockNumber>,
-    /// Current status of the vault
-    pub status: VaultStatus,
-    /// Used to calculate the amount we need to slash.
-    pub slash_per_token: SignedFixedPoint,
-    /// Updated upon deposit or withdrawal.
-    pub slash_tally: SignedFixedPoint,
-    /// Amount of collateral that is locked to back tokens. This will be
-    /// reduced if the vault has been slashed. If nomination is enabled
-    /// this will be greater than the `collateral` supplied by the vault.
-    /// Note that this excludes griefing collateral.
-    pub backing_collateral: Collateral,
```

Comment:
If a vault locks collateral in replace for griefing, is that accounted for in the `staking` pallet or not?

Reply:
It is not, we still have the `replace_collateral` field on the Vault struct for this purpose.